### ISSUE TITLE
New message: screen on

### ISSUE THREAD

**Issue by jranma** [richonguzman/LoRa_APRS_Tracker#10]
Proposal: when a new message is received, screen should turn on, so the user is notified.

Cheers 73
Julien HB9HRD

**Comment by richonguzman** [comment]
Hi!

Do you mean when screen EcoMode is active?

PS: remember to put HB9PS-10 in stationMode:2 to receive messages on your tracker ;)

**Comment by jranma** [comment]
Yes, message reception could turn on the screen when ecomode is active, this is the only way the user can get notified, i guess

**Comment by richonguzman** [comment]
nice, will look at a way to enable this

so you can choose the screen stays on then message received or not turn on until you need to
________________________________
De: jranma ***@***.***>
Enviado: viernes, 21 de julio de 2023 18:56
Para: richonguzman/LoRa_APRS_Tracker ***@***.***>
Cc: Ricardo Guzman (Richonguzman) ***@***.***>; Comment ***@***.***>
Asunto: Re: [richonguzman/LoRa_APRS_Tracker] New message: screen on (Issue #10)


Yes, message reception could turn on the screen when ecomode is active, this is the only way the user can get notified, i guess
